python/google/protobuf/internal/message_python_test.py

```diff
@@ -1,54 +0,0 @@
-#! /usr/bin/python
```

Comment:
I agree that these tests are obsolete and should be removed.

However I would like to (not necessarily in this PR) to add two tests that verify we are testing the pure-Python and the C++ implementation, respectively. But to do that we'd need a way to conditionally run either one test or the other, depending on whether --cpp_implementation was specified or not. Do you know an easy way to do that?